.github/workflows/check-available-os-versions.sh

```diff
@@ -47,11 +47,8 @@ fetch_github_runner_os_versions() {
 fetch_vmactions_os_versions() {
 	osname="$1"
 	osname_lower="$(echo ${osname} | tr '[:upper:]' '[:lower:]')"
-	fetch_webpage_as_markdown "https://github.com/vmactions/${osname_lower}-vm" \
-		| grep 'All the supported' \
-		| sed -e "s|.*${osname} ||i" -e 's|, test\.releases.*||g' -e 's|,||g' \
-		| tr ' ' '\n' \
-		| grep -v '\-pre$' \
+	fetch_webpage_as_markdown "https://raw.githubusercontent.com/vmactions/${osname_lower}-vm/refs/heads/main/conf/test.releases" \
```

Comment:
https://raw.githubusercontent.com/vmactions/freebsd-vm/refs/heads/main/conf/test.releases
that resolves, all good

> `"13.2", "13.3", "13.4", "13.5", "14.0", "14.1", "14.2", "14.3", "15.0"`